maykin_common/accounts/csrf.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.http import HttpResponseRedirect
+from django.views.csrf import (
+    CSRF_FAILURE_TEMPLATE_NAME,
+    csrf_failure as original_csrf_failure,
+)
+
+
+def csrf_failure(request, reason="", template_name=CSRF_FAILURE_TEMPLATE_NAME):
```

Comment:
```suggestion
def csrf_failure(request: HttpRequest, reason: str ="", template_name: str = CSRF_FAILURE_TEMPLATE_NAME):
```
can we type annotate by default?